common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -58,6 +58,9 @@ public class MinimapOverlay extends Overlay {
     @Persisted
     public final Config<Boolean> followPlayerRotation = new Config<>(true);
 
+    @Persisted
+    public final Config<Boolean> hideWhenUnmapped = new Config<>(true);
```

Comment:
I'm not sure this should be true by default. This was the behavior of Legacy, and always found it annoying. Even if you don't have a minimap to show, the coordinates is relevant. I saw in your demo that you kept them visible, but then it just becomes visually odd. I also think it looks a bit weird with the minimap turning on and off. Ideally, we will map most of the quest areas, so we should only rarely end up in places without a map. But that will make the minimaps disappearance even more odd-looking.

Reply:
I don't think it disappearing and reappearing is a problem, especially since Wynn has a lot of the fade to black transitions now, however the coordinates staying does look a bit weird, I've already moved my coordinates above on my instance in prep for this but I don't think we should just move it for everyone so maybe off by default is better

Reply:
add an 'inmappedarea' function and we can fix that (here or new pr)

Reply:
I don't follow, how will that fix this? I don't think we should hide coordinates when outside mapped areas if that's what you're suggesting

Reply:
oh, I thought if we hide the map, we could hide the coordinates as well.

Reply:
But I think coordinates should always be visible as that can be useful whereas a black map is pointless

Reply:
I agree that coordinates is helpful. Moving them to above the minimap could definitely be a way to make it look less odd to just have the coordinates. And maybe having a tristate for the setting could make sense, at least if bolyai thinks the coordinates should be hidden too. :)

Reply:
So, you can revert your latest fix and restore your original behavior, at least if you try to change the default positioning of the widgets so coordinates is above the minimap.

Reply:
How's this? Territory overlay should be fine as in unmapped areas there's no territories anyway
![2023-11-01_13 15 37](https://github.com/Wynntils/Artemis/assets/8337467/6ddd5e14-c308-4aca-a3b9-42b005d16834)

Reply:
I think it looks fine.